### PR TITLE
Write playbook chapter about the Learning Library

### PR DESCRIPTION
**Pull request by aldesantis**
Adds a chapter about our Learning Library to the Personal Growth section.

### CODE REVIEW

File: source/personal-growth/learning-library.html.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+learn about our recommendations for setting up a CD pipeline. Those recommendations should work in
+the vast majority of scenarios, but that doesn't mean they're always right, or that there's no
+room for iterating on them. You might want to use a different parallelization or deployment model,
+and that's perfectly fine as long as you understand both the benefits and the cost of your decision.
+
+We like to think of the Library as a starting point, not a destination. Follow the recommended
+practices if you don't know any better or have no particular reason to deviate from them. But if you
+_do_ have a reason, go ahead and enjoy the journey! 
+
+## Contributing to the Library
```
Comment (matteogalliani) [inline source/personal-growth/learning-library.html.md:59]: I feel that we should move this paragraph in the internal playbook. It doesn't add too much value here, and it is a question that might arise while consulting the library, in my opinion.

**Review by matteogalliani — DISMISSED** [review]
@aldesantis great, thanks! I've left a comment about the last section. I'm curious to know what do you think about it.

**Review by mtylty — APPROVED** [review]
❤️

File: source/personal-growth/learning-library.html.md (hunk 1 above)
Comment (aldesantis) [reply source/personal-growth/learning-library.html.md]: Done!

**Pull request merged**